com.unity.cinemachine/Runtime/Input/CinemachineDefaultInputActions.inputactions

```diff
@@ -311,6 +320,28 @@
                     "action": "Aim",
                     "isComposite": false,
                     "isPartOfComposite": false
+                },
+                {
+                    "name": "",
+                    "id": "41a57729-3be5-47fd-84d3-35eeb70d1c7e",
+                    "path": "<Keyboard>/escape",
+                    "interactions": "",
+                    "processors": "",
+                    "groups": "Keyboard&Mouse",
+                    "action": "Cancel",
+                    "isComposite": false,
+                    "isPartOfComposite": false
+                },
+                {
+                    "name": "",
+                    "id": "4fe574e1-120d-4638-8f1c-ea852a687eae",
+                    "path": "<Gamepad>/start",
+                    "interactions": "",
+                    "processors": "",
+                    "groups": "Gamepad",
+                    "action": "Cancel",
+                    "isComposite": false,
+                    "isPartOfComposite": false
```

Comment:
Cancel mapped to Escape key on keyboard and Start key on gamepad.

I choose Start key because that's usually the equivalent to Escape. Also checked if I remembered correctly, and found this as reference: https://gaming.stackexchange.com/questions/97795/what-are-the-equivalent-pc-keyboard-keys-of-the-xbox-360-game-controller-buttons.